tools/androidctl/__main__.py

```diff
@@ -89,4 +96,9 @@ def parse_cli(*args, **kwargs):
 if __name__ == '__main__':
   args = parse_cli()
   params = [getattr(args, param) for param in args.params if hasattr(args, param)]
-  args.fn(*params)
+  try:
+    args.fn(*params)
+    sys.exit(0)
+  except errors.BaseError as err:
+    sys.stderr.write('%s\n' % err.message)
+    sys.exit(err.code)
```

Comment:
once we raise an error it will catch it here and print the error message + stop the application with the exit code